Lib/test/test_configparser.py

```diff
@@ -720,6 +721,11 @@ def test_read_returns_file_list(self):
         parsed_files = cf.read(file1)
         self.assertEqual(parsed_files, [file1])
         self.assertEqual(cf.get("Foo Bar", "foo"), "newbar")
+        # check when we pass only a Path object:
+        cf = self.newconfig()
+        parsed_files = cf.read(pathlib.Path(file1))
+        self.assertEqual(parsed_files, [pathlib.Path(file1)])
```

Comment:
I only noticed `bytes` paths did weird things when checking to see why Path objects were failing. It's not something I actually make use of personally but the behaviour was unexpected. My only real argument for supporting `bytes` paths is that you already do so in lists.

I'd err on the side of not changing current (working) behaviour and leave the path objects as is.